### PR TITLE
Update docs with extra details

### PR DESCRIPTION
**Pull request by chvainickas**
# ✨ Codu Pull Request 💻

Fixes #(issue)

## Pull Request details

- INFO ABOUT YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible) 🤜
- Use bullet points to make it easy to read.

## Any Breaking changes

- IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
- IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'

## Associated Screenshots

- IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE. _( Welcome file extensions include gifs/png screenshots of your feature in action )_
- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ We have a fantastic community growing on Discord. Click [here](https://discord.g
 4. Install dependencies with:
 
 ```bash
-npm install
+npm install && npm audit fix --force
```
Comment (John-Paul-Larkin) [inline README.md]: Hey Edward,
We dont want to add "run audit fix" to the readme, since it will update packages.

Ideally this will be done as a separate PR, and the changes will be pushed to everyone. 
I ran this locally myself during the stream, and it is updating sentry, so we will need to do a little investigating on this first. Thanks

Comment (chvainickas) [reply README.md]: Thank you JP, this is a good point! I will omit this from the final doc

File: README.md [hunk 2]
```diff
@@ -45,15 +44,18 @@ For a more detailed how to guide on setting them up [go to the Environment Varia
 
 6. [Make sure your database is running](#database_url) and setup the tables in the database with Drizzle by running:
 
+
 ```bash
 npm run db:push
 ```
+The full command can be seen in our [here](/package.json#L16) in our [package.json](/package.json) file
```
Comment (John-Paul-Larkin) [inline README.md]: Hey Edward. This markdown shows as:
The full command can be seen in our here in our package.json file

Maybe remove "in our here", so it appears as :
The full command can be seen in our package.json file

This applies to the link below also.

Comment (chvainickas) [reply README.md]: Hi JP, yes, good spot! I'll change that before the final PR. Should be done by today.

**Comment by chvainickas** [comment]
Hi guys, I've finished the README.md 

I did not really need to add anything or delete anything, just kind of created more links for anyone that might want to see more under the hood of what is happening and added excerpts to be copied for OAuth, might just quicken the process for people getting set-up. 

Anyhow, I look forward to your reviews

**Review by NiallJoeMaher — APPROVED** [review]
